### ISSUE TITLE
problem using with most recent ember-cli

### ISSUE THREAD

**Issue by shaunc** [ember-cli/ember-cli-chai#4]
I get the following error when installed with HEAD version of ember-cli:

```
version: 1.13.8-master-64d729898b
undefined is not a function
TypeError: undefined is not a function
    at Class.module.exports.included (/Volumes/Macintosh_HD/Users/shauncutts/src/ember-grid/node_modules/ember-cli-chai/index.js:9:15)
    at /Volumes/Macintosh_HD/Users/shauncutts/src/ember-cli/lib/models/addon.js:241:32
    at Array.map (native)
    at Class.eachAddonInvoke (/Volumes/Macintosh_HD/Users/shauncutts/src/ember-cli/lib/models/addon.js:239:22)
    at Class.Addon.included (/Volumes/Macintosh_HD/Users/shauncutts/src/ember-cli/lib/models/addon.js:345:8)
    at EmberAddon.<anonymous> (/Volumes/Macintosh_HD/Users/shauncutts/src/ember-cli/lib/broccoli/ember-app.js:317:15)
    at Array.filter (native)
    at EmberAddon.EmberApp._notifyAddonIncluded (/Volumes/Macintosh_HD/Users/shauncutts/src/ember-cli/lib/broccoli/ember-app.js:312:45)
    at EmberAddon.EmberApp [as appConstructor] (/Volumes/Macintosh_HD/Users/shauncutts/src/ember-cli/lib/broccoli/ember-app.js:106:8)
    at new EmberAddon (/Volumes/Macintosh_HD/Users/shauncutts/src/ember-cli/lib/broccoli/ember-addon.js:39:8)
```

**Comment by shaunc** [comment]
NB I'm using in an addon. May be related to:

https://github.com/ember-cli/ember-cli/issues/3718

**Comment by Turbo87** [comment]
this should be fixed now in v0.1.0
